terraform/production/main.tf

```diff
@@ -11,8 +11,8 @@ terraform {
   }
 
   backend "s3" {
-    bucket  = "cg-220d7939-3fdc-474f-905f-cd254dd1219a"
-    key     = "terraform.tfstate.production"
+    key     = "terraform.tfstate.prod"
```

Comment:
The keys could also be a part of the vars we generate in CI, thats probably the last thing we'd need to do, at least to the `backend`, to de duplicate these files.